src/components/common/Header/Header.jsx

```diff
@@ -19,17 +22,35 @@ const Header = () => {
   const handleContact = () => {
     setIsContactOpen(!isContactOpen);
   };
+
+  const handleGoBack = () => {
+    history.goBack();
+  };
+
```

Comment:
is this function needed just for calling another function? WDYT about just using `history.goBack()` instead of `handleGoBack()` ?